src/components/ConnectWallet/ConnectWalletModal.tsx

```diff
@@ -165,6 +165,23 @@ export const DisconnectWalletModal: FC<DisconnectWalletModalProps> = ({
           </Link>
         </div>
       </div>
+      <div className="w-7/8 mx-10 mt-10 text-justify text-sm">
+        By connecting your wallet to Fortress finance, you acknowledge that you
+        have read and understand the Fortress{" "}
```

Comment:
This doesn't really make sense as a sentence I don't think: "You have read and understand the Fortress protocol".

I think it should say "you acknowledge that you have [read the documentation](link) and understand the Fortress..."

Not blocking because I'm pretty sure this is the approved copy but just my two cents.

Reply:
Makes sense - updated